src/request/import/ChooseKeyType.js

```diff
@@ -54,8 +54,11 @@ class ChooseKeyType extends Nimiq.Observable {
 
         /* eslint-disable max-len */
         $el.innerHTML = `
+            <div class="page-header">
             <h1 data-i18n="choose-key-type-heading">Choose key type</h1>
-            <h2 data-i18n="choose-key-type-subheading">We couldn't determine the type of your key. Please select it below.</h2>
+            </div>
+            <div class="page-body">
+            <p data-i18n="choose-key-type-text">We couldn't determine the type of your key. Please select it below.</p>
```

Comment:
This text is probably not very good in explaining what is happening here, but we leave it to the UX/UI designers to come up with good wording.